### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by raproid**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,15 @@
-Insolar documentation sources
+# Insolar documentation sources
+
+## Install
+1. Install the latest Sphinx version: `python3 -m pip install sphinx`
+2. Set PATH if you installed it with the `--user` key: `PATH="/Users/<user>/Library/Python/<version>/bin:$PATH"`
+3. Install the extensions used in this project: 
+   ```
+   python3 -m pip install sphinx-tabs
```
Comment (sofiayem) [inline README.md]: Make it a single command: python3 -m pip install sphinx-tabs sphinxcontrib.plantuml sphinx_copybutton sphinxcontrib.contentui

File: README.md [hunk 2]
```diff
@@ -1 +1,15 @@
...
+   python3 -m pip install sphinxcontrib.plantuml
+   python3 -m pip install sphinx_copybutton 
+   python3 -m pip install sphinxcontrib.contentui
+   ```
+## Build
+1. Go to the source files directory: `cd <local-path-to-the-repo>/docs/`
+2. Build artifacts: `make html` for HTML artifacts or `make pdf` for PDF artifacts.
```
Comment (sofiayem) [inline README.md]: Command for PDF is `make latexpdf`

**Review by sofiayem — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (raproid) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (raproid) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -1 +1,9 @@
-Insolar documentation sources
+# Insolar documentation sources
+
+## Install
+1. Install the latest Sphinx version: `python3 -m pip install sphinx`
+2. Set PATH if you installed it with the `--user` key: `PATH="/Users/<user>/Library/Python/<version>/bin:$PATH"`
+3. Install the extensions used in this project: `python3 -m pip install sphinx-tabs sphinxcontrib.plantuml sphinx_copybutton sphinxcontrib.contentui`
+## Build
+1. Go to the source files directory: `cd <local-path-to-the-repo>/docs/`
+2. Build artifacts: `make html` for HTML artifacts or `make latexpdf` for PDF artifacts.
```
Comment (sofiayem) [inline README.md]: Forgot to mention that `make latexpdf` requires [LaTeX](https://www.latex-project.org/get/).

**Review by sofiayem — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (raproid) [reply README.md]: added

**Review by sofiayem — APPROVED** [review]

**Pull request merged**